packages/block-library/src/image/edit.native.js

```diff
@@ -142,11 +143,14 @@ export class ImageEdit extends React.Component {
 			requestImageUploadCancelDialog( attributes.id );
 		} else if ( attributes.id && ! isURL( attributes.url ) ) {
 			requestImageFailedRetryDialog( attributes.id );
+		} else if ( Platform.OS === 'android' ) {
```

Comment:
Blocking in javascript to be android only until iOS native solution is complete.